images/prow-tests/Dockerfile

```diff
@@ -61,6 +61,9 @@ ADD . /go/src/knative.dev/test-infra
 RUN make -C /go/src/knative.dev/test-infra/tools/githubhelper
 RUN cp /go/src/knative.dev/test-infra/tools/githubhelper/githubhelper .
 
+# We would prefer to install with go get -u github.com/boz/kail, but it is broken
+RUN /go/src/knative.dev/test-infra/images/prow-tests/install-kail.sh
```

Comment:
I think you don't need an extra script for that. Please prove me wrong. Or not.

Reply:
Fix the dependencies, use "go get", see my comment in the main thread.

Reply:
I already did in our chat